Resources/Prototypes/DeltaV/Entities/Clothing/Ears/headsets.yml

```diff
@@ -26,7 +26,7 @@
     sprite: DeltaV/Clothing/Ears/Headsets/service.rsi
   - type: Clothing
     sprite: DeltaV/Clothing/Ears/Headsets/service.rsi
-    
+
```

Comment:
AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA

Reply:
doing us all a favor by removing these unneeded, unconventional spaces (I did not do this but I'll fix it) B)